socket-server/index.js

```diff
@@ -0,0 +1,13 @@
+var notificationHub = require('notification/hub');
+
+module.exports = function (httpServer) {
+  var io = require('socket.io')(httpServer);
+
+  io.of('/notify').on('connection', function (socket) {
```

Comment:
I normally keep a reference to the namespace when creating one as the namespace object has plenty of useful methods (especially server-side) when managing multiple clients.
``` javascript
const nsp = io.of('/notify');
nsp.on('connection', function (socket) {
```